src/commands/createWebApp/readAzConfig.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { workspace } from 'vscode';
+import { IAppServiceWizardContext } from 'vscode-azureappservice';
+import { cpUtils } from '../../utils/cpUtils';
+
+/**
+ * Takes any number of Azure config properties to read and returns their configured values.
+ * Azure CLI configuration docs: https://aka.ms/AA64syh
+ */
+export async function readAzConfig(wizardContext: IAppServiceWizardContext, ...propertiesToRead: AzConfigProperties[]): Promise<AzConfig> {
+    const config: AzConfig = {};
+    const azCommandCheck: cpUtils.ICommandResult = await cpUtils.tryExecuteCommand(undefined, undefined, 'az --version');
+
+    if (azCommandCheck.code === 0) {
+        const workspacePath: string | undefined = workspace.workspaceFolders ? workspace.workspaceFolders[0].uri.fsPath : undefined;
```

Comment:
Unless I'm missing something, that logic always results in undefined in the context of basic create

Reply:
It's only relevant when you do basic create _during_ a deploy

But also that code is supposed to handle the case when it's undefined, and I'm not sure why it's not - which is what I was asking @nturinski about

Reply:
For some reason, I removed that code in this PR: https://github.com/microsoft/vscode-azureappservice/commit/2aeb49340e4a90154e7e16d9f1bbed41cacb1571

Not entirely sure what I was thinking 😓 

As for it always being undefined, it would have a value if the user started with "Deploy" and then clicked `+ Create Web App...` when prompted for a web app.